src/clr-angular/data/datagrid/datagrid-filter.ts

```diff
@@ -71,6 +75,9 @@ export class ClrDatagridFilter<T = any> extends DatagridFilterRegistrar<T, ClrDa
     if (boolOpen !== this._open) {
       this._open = boolOpen;
       this.openChanged.emit(boolOpen);
+      if (!boolOpen) {
+        this.anchor.nativeElement.focus();
```

Comment:
It is unlikely but possible this could run in SSR, so we should wrap it in a platform check.

Reply:
Done!